Swime.xcodeproj/project.pbxproj

```diff
@@ -1,2054 +1,1351 @@
+		OBJ_149 /* Equal.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_85 /* Equal.swift */; };
+		OBJ_150 /* HaveCount.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_86 /* HaveCount.swift */; };
+		OBJ_151 /* Match.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_87 /* Match.swift */; };
+		OBJ_152 /* MatchError.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_88 /* MatchError.swift */; };
+		OBJ_153 /* MatcherFunc.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_89 /* MatcherFunc.swift */; };
+		OBJ_154 /* MatcherProtocols.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_90 /* MatcherProtocols.swift */; };
+		OBJ_155 /* PostNotification.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_91 /* PostNotification.swift */; };
+		OBJ_156 /* Predicate.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_92 /* Predicate.swift */; };
+		OBJ_157 /* RaisesException.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_93 /* RaisesException.swift */; };
+		OBJ_158 /* SatisfyAllOf.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_94 /* SatisfyAllOf.swift */; };
+		OBJ_159 /* SatisfyAnyOf.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_95 /* SatisfyAnyOf.swift */; };
+		OBJ_160 /* ThrowAssertion.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_96 /* ThrowAssertion.swift */; };
+		OBJ_161 /* ThrowError.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_97 /* ThrowError.swift */; };
+		OBJ_162 /* ToSucceed.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_98 /* ToSucceed.swift */; };
+		OBJ_163 /* Await.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_100 /* Await.swift */; };
+		OBJ_164 /* Errors.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_101 /* Errors.swift */; };
+		OBJ_165 /* Functional.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_102 /* Functional.swift */; };
+		OBJ_166 /* SourceLocation.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_103 /* SourceLocation.swift */; };
+		OBJ_167 /* Stringers.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_104 /* Stringers.swift */; };
+		OBJ_174 /* Package.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_105 /* Package.swift */; };
+		OBJ_180 /* Behavior.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_21 /* Behavior.swift */; };
+		OBJ_181 /* Callsite.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_22 /* Callsite.swift */; };
+		OBJ_182 /* Configuration.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_24 /* Configuration.swift */; };
+		OBJ_183 /* QuickConfiguration.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_25 /* QuickConfiguration.swift */; };
+		OBJ_184 /* DSL.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_27 /* DSL.swift */; };
+		OBJ_185 /* World+DSL.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_28 /* World+DSL.swift */; };
+		OBJ_186 /* ErrorUtility.swift in Sources */ = {isa = PBXBuildFile; fil
```

Comment:
@sendyhalim Since this change I've been having trouble building with an app that targets iOS 10.0. Was just curious what changed that required the bump to iOS 11.0?